src/main/java/uk/gov/hmcts/darts/usermanagement/mapper/impl/UserAccountMapper.java

```diff
@@ -40,17 +37,17 @@ public interface UserAccountMapper {
         @Mapping(source = "userName", target = "fullName"),
         @Mapping(source = "emailAddress", target = "emailAddress"),
         @Mapping(source = "userDescription", target = "description"),
-        @Mapping(source = "state", target = "state"),
+        @Mapping(source = "active", target = "state"),
         @Mapping(source = "lastLoginTime", target = "lastLogin")
     })
     UserWithIdAndLastLogin mapToUserWithIdAndLastLoginModel(UserAccountEntity userAccountEntity);
 
-    default UserState mapToUserState(Integer stateValue) {
-        return stateValue == null || ENABLED.getId().equals(stateValue) ? UserState.ENABLED : UserState.DISABLED;
+    default UserState mapToUserState(boolean stateValue) {
+        return stateValue ? UserState.ENABLED : UserState.DISABLED;
```

Comment:
i dont think we should be using text to represent a boolean.
MY feeling is that we should be changing the response of the api to `active` instead of `state`.
i am aware that this will cause a front end change, but i think it would be better.
@hemantasharma1129  thoughts?

Reply:
Agree with @cmagowan-hmcts. We can add a boolean field in Userstate.